addons/website_crm_partner_assign/views/website_crm_partner_assign_templates.xml

```diff
@@ -228,8 +228,10 @@
                 <thead>
                     <tr>
                         <th>Date</th>
-                        <th class="w-75">Name</th>
+                        <th class="w-75">Lead</th>
```

Comment:
Side note: commit header should include a verb (to be valid when making a sentence like 'if applied, this commit will ...'). Like "add lead phone and email on portal page".